cypress/e2e/WebInterface/Test Cases/QDM Test Case/QDMTestCaseCreation.cy.ts

```diff
@@ -565,5 +564,24 @@ describe('Validating Expansion -> Manifest selections / navigation functionality
 
         //verify the results row
         cy.get(TestCasesPage.testCaseResultrow).should('contain.text', 'FailQDMManifestTCGroupQDMManifestTCQDMManifestTCSelect')
+
     })
+
+    it('Verify Expansion -> Manifest: When code does not exist on value set, test case will fail. When value set does contain code, and all other expected equals actual then test case passes.', () => {
```

Comment:
Why is this test case in QDMTestCaseCreation.cy.ts file? Shouldn't it be part of [Measure/CQLEditor](https://github.com/MeasureAuthoringTool/madie-cypress/tree/master/cypress/e2e/WebInterface/Measure/QDM%20CQL%20Editor) I'm new here. ignore this if doesn't make sense.

Reply:
I included the this test within the QDMTestCaseCreation test file because that is where I placed the other manifest related automated test. I saw this as being related to manifest and that is why I just added it to this file. I can move it -- it isn't a huge deal to do that but that is my justification for putting it within this test file, atm.

Reply:
I was just curious. I don't have a strong opinion about it, you can go with whatever makes sense. having said that, this ticket has nothing to do with manifest. it is very common for a user to enter a version for value set. also, the same change applies to the library CQL editor as well.

Reply:
Yea -- after thinking about this, more, you are correct. Adding it to the QDM CQL Editor test file does make more sense. I have made this change and I have pushed it to this PR.